src/Console/JWTGenerateCertCommand.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace PHPOpenSourceSaver\JWTAuth\Console;
+
+use Illuminate\Console\Command;
+
+class JWTGenerateCertCommand extends Command
+{
+    use EnvHelperTrait;
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'jwt:generate-certs 
+        {--force : Override certificates if existing} 
+        {--algo= : Algorithm (rsa/ec)} 
+        {--bits= : RSA-Key length (1024,2048,4096,8192} 
+        {--sha= : SHA-variant (1,224,256,384,512)} 
+        {--dir= : Directory where the certificates should be placed} 
+        {--curve= : EC-Curvename (e.g. secp384r1, prime256v1 )}
+        {--passphrase= : Passphrase}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generates a new cert pair';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $force = $this->option('force');
+        $directory = $this->option('dir') ? $this->option('dir') : 'storage/certs';
+        $algo = $this->option('algo') ? $this->option('algo') : 'rsa';
+        $bits = $this->option('bits') ? intval($this->option('bits')) : 4096;
+        $shaVariant = $this->option('sha') ? intval($this->option('sha')) : 512;
+        $passphrase = $this->option('passphrase') ? $this->option('passphrase') : null;
```

Comment:
Not sure if providing the passphrase via command line is a good idea (think about shell history file).

Maybe it should be possible to `ask()` for this too?

E.g. `--passphrase` without providing one would ask for it.

Reply:
I have changed this in https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/146